chart/epinio/templates/NOTES.txt

```diff
@@ -1,8 +1,8 @@
 To interract with your Epinio installation download the latest epinio binary from https://github.com/epinio/epinio/releases/latest.
 
-Update the api location and credentials with:
+Login to the cluster with:
     
-    `epinio settings update`
+    `epinio login -u admin -p password https://epinio.{{ .Values.global.domain }}`
```

Comment:
Unless the user set these values: https://github.com/epinio/helm-charts/blob/5d947c0b9c26c2b2254ee311d0d4d96ced0291c7/chart/epinio/values.yaml#L60-L64

Maybe we should say something like:

```
`epinio login -u {{.Values.api.username}} -p api_password_here https://epinio.{{ .Values.global.domain }}`

If you didn't specify a password the default one is `password`.
```

Reply:
What about not having the password at all as a flag? In that case the user will be prompted for it
```
`epinio login -u {{.Values.api.username}} https://epinio.{{ .Values.global.domain }}`

If you didn't specify a password the default one is `password`.
```
 i.e.:
```
-> % epinio login -u admin https://epinio.172.19.0.4.omg.howdoi.website

🚢  Login to your Epinio cluster [https://epinio.172.19.0.4.omg.howdoi.website]
Password: 

⚠️  Certificate signed by unknown authority

|     KEY     |       VALUE       |
|-------------|-------------------|
| Issuer Name | CN=epinio-ca      |
| Common Name | epinio-ca         |
| Expiry      | 2022-September-01 |

Do you want to trust it (y/n): y

✔️  Trusting certificate...

✔️  Login successful
```

Reply:
yes, that would do.